src/ui/screens/Parachains/OverviewScreen.test.tsx

```diff
@@ -0,0 +1,30 @@
+import {NavigationProp} from '@react-navigation/native';
+import {ParachainsStackParamList} from '@ui/navigation/navigation';
+import React from 'react';
+import {ReactTestInstance} from 'react-test-renderer';
+import {render, waitFor, within} from 'src/testUtils';
+import {ParachainsOverviewScreen} from './OverviewScreen';
+
+let navigation: NavigationProp<ParachainsStackParamList>;
+
+describe('OverviewScreen component', () => {
+  it('should render the loading view when data is fetching', async () => {
+    const {getByTestId} = render(<ParachainsOverviewScreen navigation={navigation} />);
+    expect(getByTestId('loading_view')).toBeTruthy();
+  });
+
+  it('render the component when data arrives', async () => {
+    const {getAllByTestId, getAllByText} = await waitFor(() =>
+      render(<ParachainsOverviewScreen navigation={navigation} />),
+    );
+    const parachainItems = getAllByTestId('parachain_items');
```

Comment:
Let's not use testID, you have the mocked data so you can make assertions on the data that the user should see (as part of a parachain item)